packages/app/src/views/Sandbox.vue

```diff
@@ -13,15 +13,15 @@
     <div class="p-2">
       <!-- Dropdown -->
       <h1 class="text-2xl text-center bg-secondary py-1">Dropdown</h1>
-      <Dropdown
-        v-model="dropdownValue"
-        class="mb-5 h-20"
-        label="NFC-Tag"
-        enable-add-item
-        :items="dropdownItems"
-        @add-item="addDropdownItem"
-        @remove-item="removeDropdownItem"
-      />
+      <LabeledInput label="NFC-Tag">
+        <Dropdown
+          v-model="dropdownValue"
+          class="mb-5 h-20"
+          enable-add-item
+          :items="dropdownItems"
+          @add-item="addDropdownItem"
+          @remove-item="removeDropdownItem"
+      /></LabeledInput>
```

Comment:
this looks very weird in the sandbox. What's the intention here?
![image](https://user-images.githubusercontent.com/40185075/104716192-dc855a00-5727-11eb-836e-48e37dd7f3be.png)

Reply:
I haven't checked that. Seems like the `classes` have to go on the `LabeledInput`

Reply:
fixed by 3bc4aa4784c9286bb9085de8d47b9887d745e59b